src/eShop.AppHost/Program.cs

```diff
@@ -99,6 +99,10 @@
            .WithEnvironment("WebhooksWebClient", webhooksClient.GetEndpoint(launchProfileName))
            .WithEnvironment("WebAppClient", webApp.GetEndpoint(launchProfileName));
 
+// Starting in Aspire 9.2, we can use the new DockerComposePublisher to generate a docker-compose file.
+// In order to do so, run 'dotnet run --publisher docker-compose --output-path ./docker-compose' to try it out.
+builder.AddDockerComposePublisher();
```

Comment:
Did you test this?

Reply:
Yup, it generates the required files but when I tried doing docker compose up -d, things would run but couldn’t get https working correctly. I will look into this a bit more tomorrow.

Reply:
That's expected right as once you're not running development the dev cert won't be loaded. I thought we made the docker publisher remove the https endpoints for this reason?